### PR TITLE
Release/0.1.1

### PR DESCRIPTION
**Pull request by mhadam**
Fixes the resources folder bug, fixes the build process and fixes the README formatting.

### CODE REVIEW

**Comment by alexanderdean** [comment]
Adding @chuwy as a reviewer

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,9 @@
+Version 0.2.0 (2018-02-06)
```
Comment (chuwy) [inline CHANGELOG:1]: Don't forget to update date.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-| **[Technical Docs] [techdocs]**     | **[Setup Guide] [setup]**     | **[Roadmap] [roadmap]**           | **[Contributing] [contributing]**           |
-|-------------------------------------|-------------------------------|-----------------------------------|---------------------------------------------|
-| [![i1] [techdocs-image]] [techdocs] | [![i2] [setup-image]] [setup] | [![i3] [roadmap-image]] [roadmap] | [![i4] [contributing-image]] [contributing] |
+|  **[Technical Docs][techdocs]**   |  **[Setup Guide][setup]**   |     **[Roadmap][roadmap]**      |     **[Contributing][contributing]**      |
+|:---------------------------------:|:---------------------------:|:-------------------------------:|:-----------------------------------------:|
+| [![i1][techdocs-image]][techdocs] | [![i2][setup-image]][setup] | [![i3][roadmap-image]][roadmap] | [![i4][contributing-image]][contributing] |
 
 ## Copyright and license
 
 The Snowplow Iglu Obj-C Client is copyright 2015 Snowplow Analytics Ltd.
```
Comment (chuwy) [inline README.md:21]: Could you please also update copyright years?

**Review by chuwy — APPROVED** [review]
Overall LGTM, but I'm not sure why commit for #7 has such a huge diff. Can/should we split it to different tickets?

**Comment by alexanderdean** [comment]
Thanks @chuwy. @jbeemster, @mhadam - does the commit for #7 need to be so far-reaching, per @chuwy's question?

**Comment by jbeemster** [comment]
Yes it does - its the Pod details updating which is why there are so many file changes.

**Review by jbeemster — APPROVED** [review]

**Comment by jbeemster** [comment]
I would argue this is a bug-fix (patch release) rather than a new minor release... maybe 0.1.1 is a better Version for this?

**Comment by alexanderdean** [comment]
Renamed the milestone! Sorry for the ping-pong. @mhadam can you update the PR?

**Pull request closed without merging**